src/main/scala/it/pagopa/interop/authorizationprocess/common/AuthorizationUtils.scala

```diff
@@ -0,0 +1,35 @@
+package it.pagopa.interop.authorizationprocess.common
+
+import cats.implicits._
+import it.pagopa.interop.commons.utils.AkkaUtils.getOrganizationIdFutureUUID
+import it.pagopa.interop.authorizationprocess.error.AuthorizationProcessErrors._
+import java.util.UUID
+import scala.concurrent.{ExecutionContext, Future}
+import it.pagopa.interop.authorizationprocess.service.AuthorizationManagementService
+
+object AuthorizationUtils {
+
+  def assertIsConsumer(clientId: UUID)(
```

Comment:
just minor suggestion
```suggestion
  def assertIsClientConsumer(clientId: UUID)(
```